src/ui/public/vis/request_handlers/courier.js

```diff
@@ -46,7 +46,7 @@ const CourierRequestHandlerProvider = function () {
         col.aggConfig.isFilterable()
         && (!field || field.filterable);
       return ({
-        name: col.title,
+        name: col.name,
```

Comment:
I think the call to `tabifyAggResponse` a couple of lines above still has a couple of now non existing parameters (`canSplit`, `asAggConfigResults`) that needs to be removed.